src/fonts/OTF/TeX/Makefile

```diff
@@ -38,10 +38,7 @@ pfa: $(MFTRACE_MODIFIED)
 	$(PYTHON) $(MFTRACE_MODIFIED) --magnification 1000 --encoding $(TETEXENCODING)/10037936.enc --simplify cmsy10
 
 	@echo "cmex10"
-	$(PYTHON) $(MFTRACE_MODIFIED) --magnification 1000 --keep cmex10
-	$(FONTFORGE) -lang=ff -c 'Open("mftrace.dir/cmex10.pfa.raw");SelectAll();AddExtrema();Simplify();AutoHint();Generate("cmex10.pfa");'
```

Comment:
It looks like we were doing some hinting here, but we use `TTFAUTOHINT` below so I assume this hasn't been necessary for some time.

Reply:
@kevinbarabash `--simplify` flag to `mftrace` is equivalent to `AddExtrema();Simplify();RoundToInt();AutoHint();`. The only difference being `RoundToInt()`, I guess this was to avoid it. However, it seems there is no reason not to do it.